psl-core/src/main/java/org/linqs/psl/reasoner/sgd/SGDExtension.java

```diff
@@ -0,0 +1,46 @@
+/*
```

Comment:
Use an inner static enum instead of a different class file.
See:
https://github.com/linqs/psl/blob/develop/psl-core/src/main/java/org/linqs/psl/reasoner/admm/term/ADMMObjectiveTerm.java#L56

Reply:
Same with SGDLearningSchedule.